models/bluerov2/model.sdf

```diff
@@ -21,28 +21,26 @@
         </inertia>
       </inertial>
 
-      <visual name="base_link_box">
-        <pose>0 0 0 0 0 0</pose>
-        <transparency>0.5</transparency>
+      <visual name="base_link_visual">
+        <pose>-0.134 -0.4935 0.17 1.5707963267948966 0 1.5707963267948966</pose>
+        <geometry>
+          <mesh>
+            <uri>model://bluerov2/meshes/bluerov2-45.stl</uri>
```

Comment:
Looks like you need to re-run generate_model again.

Reply:
Sure enough :smile: